app/services/collection_space/check_media_service.rb

```diff
@@ -0,0 +1,124 @@
+module CollectionSpace
+  class CheckMediaService
+    attr_reader :client, :opts, :name, :blob
+
+    Blob = Struct.new(
+      :csid,
+      :media_csid,
+      :length,
+      :digest,
+      :name,
+      :mime_type,
+      :derivatives_count
+    )
+    DERIVABLE_IMAGE_TYPES = %w[
+      image/jpeg image/bmp image/gif image/png image/tiff application/octet-stream
```

Comment:
Remove `application/octet-stream` from this array. 

This IS the list of derivable types directly from CollectionSpace, and it does create a "content" derivative for an `application/octet-stream`. But that is different from the IMAGE derivative creation process we care about here. 

I should have included in the spec to only consider the mimetypes starting with "image/" to be derivable. My checker throws out `application/octet-stream` in its `derivable?` method, but I forgot about that.